File: Dot examples/ASP Book Examples/Book Examples/Chapter15/CS/App_Code/StoreProduct.cs
using System;

public class StoreProduct
{
    public void SaveProduct(string name)
    {
        SaveProduct(name, 0, String.Empty);
    }

    public void SaveProduct(string name, decimal price)
    {
        SaveProduct(name, price, String.Empty);
    }

    public void SaveProduct(string name, decimal price, string description)
    {
        // Save name, price, description to database
    }

}

File: Dot examples/ASP Book Examples/Book Examples/Chapter15/VB/App_Code/StoreProduct.vb
Public Class StoreProduct

    Public Sub SaveProduct(ByVal name As String)
        SaveProduct(name, 0, String.Empty)
    End Sub

    Public Sub SaveProduct(ByVal name As String, ByVal price As Decimal)
        SaveProduct(name, price, String.Empty)
    End Sub

    Public Sub SaveProduct(ByVal name As String, ByVal price As Decimal, ByVal description As String)
        ' Save name, price, description to database
    End Sub

End Class
